components/chat/messages/confirmForm.tsx

```diff
@@ -34,7 +33,7 @@ const ConfirmForm = ({
   };
 
   if (loading) {
-    return <div className="p-4">Calling tool {tool}...</div>;
```

Comment:
You actually just go ahead and remove all of the logic for loading, it isn't user anywhere here in any meaningful way.

Reply:
No, we can't remove the loading stuff. It is being used here. Specifically, to return `null` so that the form goes away.

Reply:
Didn't notice that, thanks 👍🏻